intermediateGeospatialR/src/prepWorkshopFilePrep.R

```diff
@@ -14,6 +14,8 @@ baseDir <- "D:/geoSpatialCentroid/softwareCarpentry/intermediateGeospatialR"
 ## these function are not really reusable peice of code. I'm mostly using the funtion 
 ## structure for the storage of the work. That way this script can we much cleaner looking 
 
+##EDIT: geoSpatialCentroid folder not in repo, not sure if wanting to include to walkthrough
+##      what the function is doing or just talk it through during the lesson
```

Comment:
this is primarily meant for individuals to engage with on their own time, so I should improve the commenting